src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -55,6 +61,140 @@
         Locale.ENGLISH,
     };
 
+    private static final Map<String, Set<String>> TOKENS_IN_CONFIG_TO_IGNORE = new HashMap<>();
+
+    static {
+        // can't differentiate generic methods from type parameters
+        TOKENS_IN_CONFIG_TO_IGNORE.put("NoWhitespaceBefore",
+                Stream.of("GENERIC_START", "GENERIC_END").collect(Collectors.toSet()));
+        // enum constants should be uppercase
+        TOKENS_IN_CONFIG_TO_IGNORE.put("AbbreviationAsWordInName", Stream.of("ENUM_CONSTANT_DEF")
+                .collect(Collectors.toSet()));
```

Comment:
Done. I rewrote but I was referring to enum constants/values. I mentioned `EnumValueNameCheck` in reason.